### PR TITLE
Check login page

### PR DESCRIPTION
**Pull request by helltepakcer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sudo: required
 
 addons:
   chrome: stable
+  artifacts: true
```
Comment (Anedchenko) [inline .travis.yml]: Artifacts in Travis are impossible to implement at the moment, so there's no need in this addon

File: pages/login_page.py [hunk 2]
```diff
@@ -63,4 +63,4 @@ def click_on_backspace_icon(self):
         self.backspace_icon().click()
 
     def check_where_bot(self):
```
Comment (Anedchenko) [inline pages/login_page.py]: Name of function should be changed, or whole function should be replaced with the built-in one

**Review by Anedchenko — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (helltepakcer) [reply .travis.yml]: You right, I decide to remove this line.

File: pages/login_page.py (hunk 2 above)
Comment (helltepakcer) [reply pages/login_page.py]: I change name of this method, thank you for review!

**Review by SergeyMoroz0703 — APPROVED** [review]

**Pull request merged**